clients/client-go/tcgithub/tcgithub.go

```diff
@@ -338,6 +338,19 @@ func (github *Github) CreateComment(owner, repo, number string, payload *CreateC
 	return err
 }
 
+// Stability: *** EXPERIMENTAL ***
+//
+// This endpoint allows to render the .taskcluster.yml file for a given event or payload.
+// This is useful to preview the result of the .taskcluster.yml file before pushing it to
+// the repository.
+//
+// See #renderTaskclusterYaml
+func (github *Github) RenderTaskclusterYaml(payload *RenderTaskclusterYmlInput) (*RenderTaskclusterYmlOutput, error) {
```

Comment:
Maybe we should add YAML to [commonInitialisms](https://github.com/taskcluster/taskcluster/blob/f81b376f030cf2f7fc5a0898529614a286b5492d/tools/jsonschema2go/text/text.go#L44)

Reply:
oh, so it is supposed to be uppercase always?

Reply:
replaced namings